bin/osg-notify

```diff
@@ -157,10 +157,19 @@ def main():
     if args.dryrun:
         print(msg)
     else:
-        session = smtplib.SMTP('localhost')
-        session.sendmail("help@opensciencegrid.org", recipients, msg)
-        session.quit()
-
+        while True:
+            verify_send = raw_input("Really send mail to {0} recipients? (y/N)".format(len(recipients)))
+            if verify_send.lower() == 'y':
+                session = smtplib.SMTP('localhost')
+                session.sendmail("help@opensciencegrid.org", recipients, msg)
+                session.quit()
+                break
+            elif verify_send.lower() == 'n' or verify_send.lower() == '':
```

Comment:
I recommend using `elif verify_send.lower() in ["n", ""]` instead to avoid the unnecessary second function call.